x/concentrated-liquidity/simulation/sim_msgs.go

```diff
@@ -158,7 +168,7 @@ func RandMsgCollectFees(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Con
 			return nil, err
 		}
 
-		remainingSwapOwnerToken0Amt = remainingSwapOwnerToken0Amt.Sub(randToken0Amt)
+		remainingSwapOwnerToken0Amt = remainingSwapOwnerToken0Amt.Sub(randToken0Amt).ToDec().TruncateInt()
```

Comment:
what's the reason we do int -> dec -> int conversion?

Reply:
i couldn't just truncate sdk.Int. but i can truncate sdk.Dec to lowest int. So i had to convert it to dec first. please lmk if there's a better way of handling this

Reply:
The result is already truncated, there is no need to turn it to a dec to re-truncate it

Reply:
oh yea that was dumb of me cause its an integer